src/main/java/Main.java

```diff
@@ -0,0 +1,22 @@
+public class Main {
+    public static void main(String[] args) {
+        String[][] inputArray = {
+                {"10", "12", "23", "14"},
+                {"65", "invalid value #1", "36", "17"},
+                {"invalid value #2", "55", "96", "67"},
+                {"51", "60", "invalid value #3", "87"}
+        };
+        int sum = 0;
+        try {
+            ArrayValueCalculator calculator = new ArrayValueCalculator();
+            sum = calculator.doCalc(inputArray);
+        } catch (ArraySizeException | ArrayDataException e) {
+            System.out.println(e.getMessage());
+        } catch (Exception e) {
```

Comment:
`catch (Exception)` and `finally` doesn't need for your task. But good that you tach it by hands!